MonoGame.Framework/Graphics/SurfaceFormat.cs

```diff
@@ -36,7 +36,7 @@ public enum SurfaceFormat
         /// <summary>
         /// DXT5. Texture format with compression. Surface dimensions must be a multiple 4.
         /// </summary>
-        Dxt5 = 6,
+        Dxt5,

```

Comment:
No functional or numeric change to the enum values...   this was just a mistake that wasn't necessary.  Makes interop with native code more clear.